avocado/utils/linux_modules.py

```diff
@@ -153,3 +159,42 @@ def module_is_loaded(module_name):
 def get_loaded_modules():
     lsmod_output = process.system_output('/sbin/lsmod').splitlines()[1:]
     return [line.split(None, 1)[0] for line in lsmod_output]
+
+
+class Config_check_mesg(Enum):
+    """
+    enum class to config option , represent in
+    running kernel
+    0 : config not avilable
+    1 : config avilable as dynamic loadable
+    2: config avilable as loaded
+    """
+    notset = 0
+    Dynamically_loadable = 1
+    loaded = 2
+
+
+def check_kernel_config(config_name):
+    """
+    Method : pass a config option and it return
+             - return enum 0 : config not  avilable
+             - return enum 1 : config as in  loadable
+             - return enum 2 : config already loaded
```

Comment:
Avocado uses Sphinx to generate the documentation, so it'd be better to follow the standard declarations. You can find some details in http://www.sphinx-doc.org/en/stable/domains.html#the-python-domain I can recommend Eclipse (others would not agree with me, though :smile:) where you just type `:p` and by pressing `ctrl+space` it offers the options.

Anyway the basics are:

```
Short description on a first line

Longer multi-line details, if necessary.
:param config_name: what this option means
:return: Description of the return, for example: It returns either NOT_SET, MODULE or BUILTIN
```
Yes, there are fancier definitions, for example the `:rtype` and defining it properly, but not even us follow that, so it's not mandatory.